packages/scrypt-kdf/scrypt-kdf.0.1.0/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "1.2"
+maintainer: "Alfredo Beaumont <alfredo.beaumont@gmail.com>"
+authors: [
+  "Alfredo Beaumont <alfredo.beaumont@gmail.com>"
+  "Sonia Meruelo <smeruelo@gmail.com>"
+]
+homepage: "https://github.com/abeaumont/ocaml-scrypt-kdf"
+bug-reports: "https://github.com/abeaumont/ocaml-scrypt-kdf/issues"
+license: "BSD2"
+dev-repo: "https://github.com/abeaumont/ocaml-scrypt-kdf.git"
+build: [
+  "ocaml"
+  "pkg/build.ml"
+  "native=%{ocaml-native}%"
+  "native-dynlink=%{ocaml-native-dynlink}%"
+  "alcotest=false"
+]
+build-test: [
+  [
+    "ocaml"
+    "pkg/build.ml"
+    "native=%{ocaml-native}%"
+    "native-dynlink=%{ocaml-native-dynlink}%"
+    "alcotest=true"
+  ]
+  ["sh" "-exc" "./run_tests.sh"]
```

Comment:
The extra `sh` invocation here seems unnecessary. Couldn't you just use `"./run_tests.sh"`? This won't block merge.